rust/agama-server/src/web/event.rs

```diff
@@ -21,6 +23,13 @@ pub enum Event {
     ProductChanged {
         id: String,
     },
+    FirstUserChanged(FirstUser),
+    RootPasswordChanged {
```

Comment:
Why not merge both root actions?

```rust
RootUserChanged: {
  sshkey: Option<String>,
  has_password: Option<bool>
}
```

Reply:
well, we can merge even all of them and have UsersChanged and have complete get content there. Or as in question simple UsersChanged and force user to reread values with get. This way it just follows dbus signals and use value it provides.